pokemon-app/src/selectors/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { createSelector} from "reselect";
+import { RootStateOrAny} from 'react-redux'
+import { Pokemon } from "../types";
+
+const selectAllPokemons = (state: RootStateOrAny) => state.pokemons
```

Comment:
Also please do not forget about semicolons. You can add a rule for your eslint to automate this process